internal/public.proto

```diff
@@ -40,6 +40,10 @@ message Bit {
 	int64 Timestamp = 3;
 }
 
+message BulkColumnAttrRequest {
```

Comment:
I don't think regenerating should have changed `private.pb.go`. It looks like is a golang or protoc version difference. Let's check with @codysoyland to be sure we're all using the same version to generate code. And if the version does need to change, let's first commit the version-specific change in its own PR.